Cargo.toml

```diff
@@ -8,9 +8,9 @@ default = ["rand_chacha"]
 
 [dependencies]
 miden-crypto = { version = "0.12", default-features = false }
-miden-lib = { version = "0.6", default-features = false }
-miden-assembly = { version = "0.11", default-features = false }
-miden-objects = { version = "0.6", default-features = false }
-miden-tx = { version = "0.6", features = ["testing"] }
+miden-lib = { git = "https://github.com/0xPolygonMiden/miden-base", branch = "andrew-rework-fpi-tests", default-features = false, features = ["concurrent", "testing"] }
```

Comment:
Thanks that you made that work! 

Let's not merge it until we have those changes in main. I want Pioneers and externals only to work on stable versions.

Reply:
Sounds good, for now the changes have been added to "next" hence re-routed the crates to "next". 

We will need to wait for the next release to get it on main.